config/metrics/metrics-prometheus-rules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: osc-alerts
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  groups:
+  - name: osc_alerts
+    rules:
+    - alert: KataRemoteWorkloadFailureHigh
+      expr: kata_remote_workload_failure_ratio > 0.25
```

Comment:
I think kata_remote_workload_failure_ratio is a full percentage already.
So that would be > 25 I guess?

Reply:
Indeed :smile:

```
	kataRemoteWorkloadFailureRatio.Set(float64(failedPods) / float64(totalPods) * 100)
```

Good catch @littlejawa !